projects/bzip2/bzip2_compress_target.c

```diff
@@ -55,7 +73,9 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 
     nOut = size*2;
     char *outbuf = malloc(nOut);
-    r = BZ2_bzBuffToBuffDecompress(outbuf, &nOut, zbuf, nZ, 0, 0);
+    small = size % 2;
+    r = BZ2_bzBuffToBuffDecompress(outbuf, &nOut, zbuf, nZ, small,
```

Comment:
actually, it would be nice to enable this decompression call in release build as well

Reply:
the same for asserts on line 88 and 89

Reply:
Hi @Dor1s I didn't follow your comment. What do you mean by a release build?

Reply:
Hi @Dor1s iirc, there was already a [bug report generated for the assertion failure](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=11091). So, I'm guessing assertions are enabled in the fuzzer targets, right?